_posts/2016-08-26-babili.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title:  "Releasing Babili (beta)"
+author: Henry Zhu
+date:   2016-08-26 09:30:00
+categories: announcements
+share_text: "Releasing Babili (beta)"
+---
+
+# Babili (babel-minify)
+
+We released [babili](https://github.com/babel/babili) as beta (0.0.1) yesterday under an MIT license!
+
+There are a lot of (valid) questions about why a new minifier is a good idea, so this post should help with that. We also have a [#minify](https://babeljs.slack.com/messages/minify/) slack room!
+
+TL;DR: Babili can output to ES2015+ code so you don't need to transpile and then minify if it's not necessary. Babili is also modular/flexible (it's a babel preset and thus supports user plugins) and can be used as a preset or cli tool. Babili should also be able to do ES2015+ specific optimizations.
```

Comment:
👍 to @loganfsmyth’s suggestion, but with this tweak: `s/mostly accept/are mostly limited to/`